libs/feature/editor/src/lib/components/contact-card/contact-card.component.html

```diff
@@ -0,0 +1,28 @@
+<ng-container *ngIf="organization && contact">
```

Comment:
I think it is fair to assume that a contact and an org will be provided for this component, so no need to have this check